src/dosbox.cpp

```diff
@@ -781,19 +781,22 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&DMA_Init);
 	secprop->AddInitFunction(&VGA_Init);
 	secprop->AddInitFunction(&KEYBOARD_Init);
+	secprop->AddInitFunction(&PCI_Init); // PCI bus
 
-	secprop=control->AddSection_prop("pci", &PCI_Init, false); // PCI bus
-
-	secprop->AddInitFunction(&VOODOO_Init, false);
+	secprop = control->AddSection_prop("voodoo", &VOODOO_Init, false);
 
 	const char* voodootypes[] = { "12mb", "4mb", "disabled", nullptr };
-	pstring = secprop->Add_string("voodoo", only_at_start, "12mb");
+	pstring = secprop->Add_string("voodoo_memsize", only_at_start, "12mb");
 	pstring->Set_values(voodootypes);
-	pstring->Set_help("RAM amount of emulated Vodooo 3dfx card.");
+	pstring->Set_help("Memory size of the emulated Vodooo 3dfx card ('12mb' by default).");
 
 	pint = secprop->Add_int("voodoo_perf", only_at_start, 0);
-	pint->SetMinMax(0, 4);
-	pint->Set_help("Toggle performance optimizations for Vodooo 3dfx emulation (0 = none, 1 = use multi-threading, 2 = disable bilinear filter, 3 = both).");
+	pint->SetMinMax(0, 3);
+	pint->Set_help("Performance optimizations avaiable when emulating the Vodooo 3dfx card:\n"
```

Comment:
typo: available

I think the word "emulate" is warranted here 😄

Reply:
But this should be reworded: "Performance optimisations to use when emulating the 3dfx Voodoo card" (not Voodoo 3dfx; 3dfx is the name of the company).

Reply:
> I think the word "emulate" is warranted here smile

yes :-) and replaced w/ your improved wording. Much better!